pkg/apis/hwameistor/v1alpha1/localdisk_types.go

```diff
@@ -65,6 +65,12 @@ type DiskAttributes struct {
 
 	// Protocol is for data transport, such as ATA, SCSI, NVMe
 	Protocol string `json:"protocol,omitempty"`
+
+	// Major represents drive used by the device
+	Major string `json:"major,omitempty"`
+
+	// Minor is used to distinguish different devices
+	Minor string `json:"minor,omitempty"`
```

Comment:
The `Major:Minor` is good for me.

BTW, the value of Major:Minor is determined by the driver, whether it is better to define it at the upper level?

Reply:
Actually, I'm confused about spec.diskAttributes and don't know when should I put attributes into it.

Reply:
`DiskAttributes` represent certain hardware/static attributes of the disk.

Some properties of the disk itself are suitable here, and these properties will not change as the system changes or administrator actions (for example, inserting or removing disks will cause DEVPATH changes).

Reply:
Got it. I'll fix this part.

Reply:
Is there any other mistakes should I fix?

Reply:
No, thank you. Once it ready, the e2e test passes and it can be merged.

Reply:
Review again, plz.

Reply:
Thanks, this is good to me. The e2e test is running now.